wcst_encode/data_utils.py

```diff
@@ -0,0 +1,128 @@
+from .constants import FEATURES
+import numpy as np
+import pandas as pd
+
+def get_behavior_by_bins(bin_size, beh):
+    """
+    bin_size: in miliseconds, bin size
+    data: dataframe for behavioral data from object features csv
+    Returns: new dataframe with one-hot encoding of features, feedback
+    """
+    max_time = np.max(beh["TrialEnd"].values)
+    max_bin_idx = int(max_time / bin_size) + 1
```

Comment:
Doesn't this overcount the bin when the edge of `max_time` aligns with integer multiples of `bin_size`? I'd think it'd make sense for this to be `ceil(max_time / bin_size)`

Reply:
ah good catch, fixed.